test/e2e/suite/trustpolicy/trusted_identity.go

```diff
@@ -76,12 +76,13 @@ var _ = Describe("notation trust policy trusted identity test", func() {
 				AddTrustPolicyOption("multiple_trusted_identity_trustpolicy.json"),
 				AddTrustStoreOption("e2e", filepath.Join(NotationE2ELocalKeysDir, "new_e2e.crt")),
 				AddTrustStoreOption("e2e", filepath.Join(NotationE2ELocalKeysDir, "e2e.crt")),
+				EnableExperimental(),
 			)
 
-			notation.Exec("verify", artifact1.ReferenceWithDigest(), "-v").
+			notation.Exec("verify", "--allow-referrers-api", artifact1.ReferenceWithDigest(), "-v").
```

Comment:
As discussed over slack please open an issue to fix this once rc5 is released.
```
Pritesh Bandi [11:34 PM] Hi Patrick - I was reviewing https://github.com/notaryproject/notation/pull/663/files#diff-719e0e8132d59c0c8f273651d38b7b382e16959ecc8de341080367a1e779bb0e
and have question on why do we need to update trust policy test to use experimental flag?

Patrick Zheng [11:42 PM]
Hi Pritesh, that's because some of those tests are expecting successful verification. Since the OldNotation is using the Referrers API by default for sign, whereas RC.5 logic is using the Referrers tag schema as default for verify. In this case, the Referrers tag schema does not exist, and thus, the test would fail with error no signature associated. To make these tests runnable again, we rather need to Sign with RC.5 notation OR Verify with the Referrers API (i.e., turning on the experimental and set --allow-referrers-api).
```

Reply:
https://github.com/notaryproject/notation/issues/668

Reply:
> As discussed over slack please open an issue to fix this once rc5 is released.
> 
> ```
> Pritesh Bandi [11:34 PM] Hi Patrick - I was reviewing https://github.com/notaryproject/notation/pull/663/files#diff-719e0e8132d59c0c8f273651d38b7b382e16959ecc8de341080367a1e779bb0e
> and have question on why do we need to update trust policy test to use experimental flag?
> 
> Patrick Zheng [11:42 PM]
> Hi Pritesh, that's because some of those tests are expecting successful verification. Since the OldNotation is using the Referrers API by default for sign, whereas RC.5 logic is using the Referrers tag schema as default for verify. In this case, the Referrers tag schema does not exist, and thus, the test would fail with error no signature associated. To make these tests runnable again, we rather need to Sign with RC.5 notation OR Verify with the Referrers API (i.e., turning on the experimental and set --allow-referrers-api).
> ```

Thanks @priteshbandi, I've alreday created an issue regarding this: https://github.com/notaryproject/notation/issues/668. We will work on it after RC.5 and before v1.0.